nixos/modules/system/boot/systemd.nix

```diff
@@ -113,7 +113,11 @@ let
       "sleep.target"
       "hybrid-sleep.target"
       "systemd-hibernate.service"
+      ] ++ optionals cfg.package.withBootloader [
       "systemd-hibernate-clear.service"
+      "systemd-bootctl@.service"
+      "systemd-bootctl.socket"
+      ] ++ [
```

Comment:
I don't think `systemd-hibernate-clear.service` is dependent on `withBootloader`.